patroni/dcs/consul.py

```diff
@@ -191,16 +191,18 @@ def touch_member(self, data, **kwargs):
             return True
 
         try:
-            self._client.kv.put(self.member_path, data, acquire=self._session)
+            args = {'cas': None} if kwargs.get('permanent') else {}
```

Comment:
The other notice is valid, but I don't get the kwargs comment. You've put it there, presumably, in order to avoid complaints about unused parameters, so now we can use it as a proper mechanism to pass the optional params. I think mixing the named param and kwargs would be ugly and confusing, since you'd assume {{permanent}} is the first optional param, while in the original function it is the second one (after the {{ttl}}).